CONTRIBUTING.md

```diff
@@ -127,6 +155,37 @@ Standard prefixes for ITK commit messages:
   * `STYLE:` No logic impact (indentation, comments)
   * `WIP:` Work In Progress not ready for merge
 
+The body of the message should clearly describe the motivation of the commit
+(**what**, **why**, and **how**). In order to ease the task of reviewing
+commits, the message body should follow the following guidelines:
+
+  1. Leave a blank line between the subject and the body.
+  This helps `git log` and `git rebase` work nicely, and allows to generate
+  smooth release notes.
+  2. Try to keep the subject line below 72 characters, ideally 50.
+  3. Capitalize the subject line.
+  4. End the subject line with a period.
```

Comment:
Per typical convention, the subject line should **not** end with a period.

Reply:
I've been adding a period since my very first commit I think :blush: . I am SORRY !  Done in 56ed51a.